app/controllers/api/v0/tasks_controller.rb

```diff
@@ -3,6 +3,7 @@ module V0
     class TasksController < ApplicationController
       include Api::V0::Mixins::IndexMixin
       include Api::V0::Mixins::ShowMixin
+      include Api::V0::Mixins::UpdateMixin
```

Comment:
This should be in the `V0x1::TasksController`, right?

Reply:
Gah! Yes, you are correct.